cmd/pebble/main.go

```diff
@@ -71,11 +72,26 @@ func main() {
 	wfeImpl := wfe.New(logger, db, va, ca, *strictMode)
 	muxHandler := wfeImpl.Handler()
 
+	if len(c.Pebble.ManagementListenAddress) > 0 {
```

Comment:
Yes I think so also. I was just following this approach because it is used on another if branch above, but it is certainly better to stick to idiomatic solutions nonetheless.

Reply:
Ahh! Apologies for missing that other bit in earlier reviews :-) Thanks for fixing this instance :+1: